sci_strategic_plugin/src/sci_strategic_plugin.cpp

```diff
@@ -810,11 +899,26 @@ cav_msgs::Maneuver SCIStrategicPlugin::composeIntersectionTransitMessage(double
 
   // Start time and end time for maneuver are assigned in updateTimeProgress
 
-  ROS_INFO_STREAM("Creating IntersectionTransitManeuver start dist: " << start_dist << " end dist: " << end_dist
+  RCLCPP_INFO_STREAM(rclcpp::get_logger("sci_strategic_plugin"), "Creating IntersectionTransitManeuver start dist: " << start_dist << " end dist: " << end_dist
                                                                       << " From lanelet: " << starting_lane_id
                                                                       << " to lanelet: " << ending_lane_id);
 
   return maneuver_msg;
 }
 
+bool SCIStrategicPlugin::get_availability()
+{
+  return true;
```

Comment:
What is this meant for?

Reply:
This is an override of base virtual class function. It has to be true by default i think, but subsystem_controllers are capable of turning it false when it is needed to turn off.
https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/carma_guidance_plugins/include/carma_guidance_plugins/plugin_base_node.hpp#L139